src/dash/DashAdapter.js

```diff
@@ -989,6 +989,16 @@ function DashAdapter() {
         }
 
         mediaInfo.isText = dashManifestModel.getIsText(realAdaptation);
+        mediaInfo.essentialProperties = dashManifestModel.getEssentialPropertiesForAdaptation(realAdaptation);
```

Comment:
Please put this into a separate function similar to `_getCommonRepresentationSupplementalProperties`